server/rpc_server.go

```diff
@@ -41,6 +55,64 @@ func NewRPCServer(security config.Security) *grpc.Server {
 	if s == nil {
 		s = grpc.NewServer()
 	}
-	diagnosticspb.RegisterDiagnosticsServer(s, &sysutil.DiagnoseServer{})
+	globalDomain = dom
+	// For redirection the cop task.
+	mocktikv.TiDBRPCServerCoprocessorHandler = HandleCopDAGRequest
+	srv := &rpcServer{}
```

Comment:
You can define a field `*domain.Domain` to avoid a global variable. and redirect with `mocktikv.TiDBRPCServerCoprocessorHandler = src.HandleCopDAGRequest`.

Reply:
Great, done.